docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -10,25 +10,40 @@
    "source": [
```

Comment:
<p>Since <code>catalog</code>,<code>flyingpigeon</code> and <code>malleefowl</code> are marked as deprecated, they will not be enabled by default. The output compare will fail in the CI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Ouranos Prod currently do not have those birds (<code>catalog</code>,<code>flyingpigeon</code> and <code>malleefowl)</code> enabled ... I wonder how they end up in this output in the first place !!!</p><p> </p><p>Is it possible that if we have them enabled in the past, but disabled now, they still persist?</p>